build/util/file_io.py

```diff
@@ -14,7 +14,7 @@ def read_from_file(file_location: Path, error_manager: InjectionErrorManager) ->
         with Path.open(file_location) as f:
             return f.readlines()
     except (FileNotFoundError, IsADirectoryError):
-        error_manager.extraInfo = file_location.as_posix()
+        error_manager.context = file_location.as_posix()
```

Comment:
I don't think `as_posix()` is cross-platform.

Reply:
`as_posix` works file on Windows. See: https://docs.python.org/3/library/pathlib.html

We're only using it to print instead of opening files through the error manager, so it should be fine to use it here.